### DISCUSSION TITLE
Multiplayer Server에서 Teleport

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1029] [5. Q&A]
Multiplayer server에서 teleport를 하면
다른 사용자 시점에서는 플레이어가 텔레포트한 위치까지 걸어가는 것처럼 보이는데

다른 사용자 시점에서도 바로 이동할 수 있게 만들 수 없을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/world-movingblock-sample 에서 상대방이 떨어질 경우
다시 시작 지점으로 텔레포트하는 코드가 구현되어 있습니다.
멀티플레이 부분 참고 부탁 드립니다.
